uwblueprint/rowan-house
# Module editor modal enhancements

## Implementation Description
- Made the edit module available from within modules themselves
- Added validation and help text 

Resolves https://github.com/uwblueprint/rowan-house/issues/88
## Screenshots

https://user-images.githubusercontent.com/36215359/172278696-04a12475-6e7f-4a55-b582-cbfd0a1edc42.mov

## What should reviewers focus on?
- Functionality of editing a module from within the module itself. Make sure you can't enter invalid text, when you save everything is properly saved, and that the help text is visible and working.

## Testing Procedure
- Click into a module 
- Edit the title and the description and save
- Make sure it is saved when you reopen the edit module or view the module itself
- Ensure that the title field is validated. That is, entering no characters or > max characters shown on help text fails.

## Things to Note
- N/A

## Checklist
